### DISCUSSION TITLE
DocumentLoaderAzureForm

### DISCUSSION THREAD

**Discussion by jmottishaw** [enoch3712/ExtractThinker#142] [Q&A]
I am unable to get DocumentLoaderAzureForm to load a stream - I suspect this may be a bug but since I can work around this tempfile I thought I would ask for some example code using BytesIO in case I am simply not doing it right.

thanks!

**Comment by enoch3712** [comment]
Hello @jmottishaw !

Yes, you are completely right. Actually none of them work, because of the recent rewrite a couple of days ago. Gonna push soon.

https://github.com/enoch3712/ExtractThinker/issues/143

Thank you and Happy Holidays!

**Comment by enoch3712** [comment]
@jmottishaw 

Fixed!

Tell me if you can check so i can close this

**Comment by jmottishaw** [comment]
thanks but I have finished my models/extractions for the moment. will refactor the code to use stream rather than annoying tempfiles after Christmas :)
